apps/graphql/package.json

```diff
@@ -25,14 +25,15 @@
     "webpack": "^4.29.1"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-decorators": "^7.4.4",
```

Comment:
Out of curiosity, why do we need this now?

Reply:
🤷‍♂ to run the gql server, it crashed without it saying it could not find it